src/Transport/Sending/DefaultBatchedOperationsToBrokeredMessagesConverter.cs

```diff
@@ -81,8 +80,16 @@ void SetReplyToAddress(OutgoingMessage outgoingMessage, BrokeredMessage brokered
 
                     if (selected != null)
                     {
-                        replyTo = mapper.Map(new EntityAddress(replyTo.Name, selected.Name));
+                        if (mapper is DefaultNamespaceNameToConnectionStringMapper)
```

Comment:
DefaultNamespaceNameToConnectionStringMapper should allow connectionstring suffxes without trying to map them, because the content of the reply to header must match the rules of the other endpoint, not the sending endpoint's, it does not necessarliy adhere to the local send/receive/partitioning rules

Imagine there are 3 endpoints, 2 belong to the same 'partitioning set' and share the notion of namespace names. A third is some V5 endpoint that needs to be sent to and is replying back, but it is hosted in some other namespace, like ServiceControl, then you need to be able to send to it directly using a connectionstring regardless of the local setting.

PassThroughNamespaceNameToConnectionStringMapper's behavior is not correct either for the reply to header when the replying endpoint is in another namespace, it does not append anything so the replyer will reply to it's own namespace, while it should reply to the namespace with current endpoint's input queue. In this case the connectionstring should be appended if the replier is V5, or if it is V6 with UseNames off.